packages/promise-cache/src/promiseCache.spec.ts

```diff
@@ -1,12 +1,18 @@
 import { beforeEach, describe, expect, it, vi } from 'vitest'
 import { PromiseCache } from './promiseCache'
 
+console.error = vi.fn()
```

Comment:
Could this affect following test files? Should it be restored `afterAll`?

Reply:
Good point, fixed!